src/components/navigation/breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,23 @@
+Use breadcrumbs to chain a list of items together
+```
+
+function Link({destination, name}){
+  return <a href={`#${destination}`} style={{textDecorationLine: 'none'}}>{name}</a>
+}
+
+function Button({action, name}){
+  return <button onClick={action}>{name}</button>

```

Comment:
It'd be cool to see our Button component in here to show the versatility (and style).